docs/advanced_usage.rst

```diff
@@ -241,6 +241,62 @@ next step of the pipeline or ``None`` if the trace should be discarded::
 
 (see filters.py for other example implementations)
 
+.. _`Logs Injection`:
+
+Logs Injection
```

Comment:
Can we import the docs from `ddtrace/contrib/logging/__init__.py` here? (not a rhetorical question)

Could be a good place to consolidate documentation. Or we could just remove from `__init__.py` too.

Reply:
Sure! Added this with the `.. automodule::` directive.